src/polyfills.ts

```diff
@@ -78,3 +78,7 @@ import 'zone.js/dist/zone';  // Included with Angular CLI.
 /***************************************************************************************************
  * APPLICATION IMPORTS
  */
+
+
+// Add global to window, assigning the value of window itself.
+(window as any).global = window;
```

Comment:
is this required?

Reply:
Yes, there is a bug with socketio and angular 6+ which this fixes https://medium.com/iamdeepinder/creating-a-real-time-app-with-angular-8-and-socket-io-with-nodejs-af63bd59a47f

Reply:
ok, we would have to include this in the ReadMe, and highlight it in the integration instructions for ICP